MacOSExample/package.json

```diff
@@ -11,18 +11,23 @@
     "test": "jest"
   },
   "dependencies": {
+    "@expo/metro-config": "^0.18.11",
+    "@expo/metro-runtime": "^3.2.3",
+    "@react-native-async-storage/async-storage": "^1.24.0",
     "@react-navigation/native": "^6.1.6",
     "@react-navigation/native-stack": "^6.9.12",
     "@react-navigation/stack": "^6.3.17",
     "hoist-non-react-statics": "^3.3.2",
     "invariant": "^2.2.4",
+    "patch-package": "^8.0.0",
+    "postinstall-postinstall": "^2.1.0",
     "react": "18.2.0",
-    "react-native": "0.72.4",
+    "react-native": "^0.74.1",
     "react-native-gesture-handler": "link:../",
-    "react-native-macos": "^0.72.0",
-    "react-native-reanimated": "^3.5.1",
-    "react-native-safe-area-context": "^4.7.2",
-    "react-native-screens": "^3.25.0"
+    "react-native-macos": "^0.73.35",
```

Comment:
Are you sure those 2 versions are compatible? As far as I can see, `react-native-macos` already has [0.74.1 version](https://github.com/microsoft/react-native-macos/releases/tag/v0.74.1)

Reply:
Of course I meant `react-native` and `react-native-macos` 😅

Reply:
> Are you sure those 2 versions are compatible?

They certainly aren't, but I think they work together because `react-native` is not used at all.

While I'm not certain about this, I believe `react-native-macos` is a complete replacement for `react-native`, as it's declared as an alias for `react-native` in the `Babel` configuration file

Reply:
removed `react-native` package in [7399ccc](https://github.com/software-mansion/react-native-gesture-handler/pull/3055/commits/7399ccc0ba8ba1230cf8c614de6b03d9ec86dc08)